packages/cli/prisma/schema.prisma

```diff
@@ -39,3 +40,21 @@ model ApiKey {
 
   @@index([organizationId])
 }
+
+model List {
+  id             String       @id @default(cuid())
+  name           String
+  organizationId String
+  Organization   Organization @relation(fields: [organizationId], references: [id])
+  Member         Member[]
+
+  @@index([organizationId])
+}
+
+model Member {
+  id     String @id @default(cuid())
```

Comment:
@elliothursh i wonder if the Send model should reference this rather than "to" as a string